doc/modules/changes.h

```diff
@@ -36,6 +37,11 @@
  * <br>
  * (Juliane Dannberg, 2016/05/19)
  *
+ * <li> New: There is now a visualization plugin to visualize the maximum
+ * horizontal component of the compressive strength.
```

Comment:
Do you mean compressive stress? I usually associate 'strength' with a plastic failure criterion.